tests/OmniSharp.Roslyn.CSharp.Tests/SemanticHighlightFacts.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Models.SemanticHighlight;
+using OmniSharp.Models.V2;
+using OmniSharp.Roslyn.CSharp.Services.SemanticHighlight;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Roslyn.CSharp.Tests
+{
+    public class SemanticHighlightFacts : AbstractSingleRequestHandlerTestFixture<SemanticHighlightService>
+    {
+        public SemanticHighlightFacts(ITestOutputHelper output, SharedOmniSharpHostFixture sharedOmniSharpHostFixture)
+            : base(output, sharedOmniSharpHostFixture)
+        {
+        }
+
+        protected override string EndpointName => OmniSharpEndpoints.V2.SemanticHighlight;
+
+        [Fact]
+        public async Task SemanticHighlightSingleLine()
+        {
+            var testFile = new TestFile("a.cs", @"
+namespace N1
+{
+    class C1 { int n = true; }
+}
+");
+
+            var line = 3;
+            var highlights = await GetSemanticHighlightsForLineAsync(testFile, line);
+
+            Console.WriteLine(string.Join("\n", highlights.Select(span => span.Type.ToString())));
```

Comment:
it's better to use `ITestOutputHelper`, it's on the base class

Reply:
Thanks, I actually should have removed that once I worked out why ranges weren't returning.